config/permissions-open.yaml

```diff
@@ -571,4 +571,4 @@ default:
 #       - name: 'Index'
 #         path: '/'
```

Comment:
Need to add App Enablement section in open & secure permissions files:
- _permissions-open.yaml:_ Add commented endpoint permissions
- _permissions-secure.yaml:_ Add default permissions to verify & allow authenticated users

Reply:
done